examples/jsm/utils/SkeletonUtils.js

```diff
@@ -447,10 +447,7 @@ function getBones( skeleton ) {
 
 function getHelperFromSkeleton( skeleton ) {
 
-	const source = new SkeletonHelper( skeleton.bones[ 0 ] );
-	source.skeleton = skeleton;
-
-	return source;
+	return new SkeletonHelper( skeleton.bones[ 0 ] );
```

Comment:
Can you please also revert these changes as well? `retargetClip()` relies on the previous code so must test the removal in `getHelperFromSkeleton()`. But let's not do this in this PR.